src/runtime/capabilities-resolver.ts

```diff
@@ -65,14 +65,12 @@ export class CapabilitiesResolver {
 
   constructor(public readonly options: CapabilitiesResolverOptions,
               creators?: StorageKeyCreatorInfo[]) {
+    this.creators = CapabilitiesResolver.getDefaultCreators();
     if (creators) {
```

Comment:
the intention of the `creators` was to override the registered creators.
also with this change (still going through my email backlog, so might be wrong) the ts and the kt capabilities resolver implementations are now different.

Reply:
I'll write a PR to fix this. Thanks for looking this over.